packages/targeting/test/targeting.test.ts

```diff
@@ -71,8 +75,103 @@ describe('targeting', () => {
           session_id: 123,
           event_types: ['Add to Cart', 'Purchase'],
         },
-        [flagConfig],
+        [flagCatchAll],
       );
     });
   });
+
+  describe('condition tests', () => {
```

Comment:
It'd be nice to create a fixture here instead. Something like I did [here](https://github.com/amplitude/Amplitude-TypeScript/blob/584e1a2aefaea14d344dcb607cc77ec518c9567c/packages/session-replay-browser/test/hooks/click.test.ts#L153). That way we can keep most of the fields that are common and we don't care too much about for a particular test the same. It makes it easier to write smaller tests more quickly. Let me know what you think!

Reply:
For sure, I definitely like that approach too, although in some cases I like to see the entirety of a config in a test data file (that was helpful to me when understanding the entirety of the flag config in the first place). I'm going to leave these as is for now but happy to iterate in that direction in a future pr